dnb.rb

```diff
@@ -0,0 +1,1451 @@
+      ]
+    end,
+    contact_organization_schema: lambda do
+      call('criteria_organization_schema').
+        only('duns', 'primaryName', 'tickerSymbol', 'primaryIndustryCodes',
+             'numberOfEmployees', 'industryCodes').
+        concat([
+                 { name: 'globalUltimate', type: 'object',
+                   properties: [
+                     { name: 'duns' }
+                   ] },
+                 { name: 'primaryAddress', type: 'object',
+                   properties: call('address_schema') },
+                 { name: 'dunsControlStatus', type: 'object',
+                   properties: [
+                     { name: 'operatingStatus', type: 'object',
+                       properties: [
+                         { name: 'dnbCode', type: 'integer' },
+                         { name: 'description' }
+                       ] },
+                     { name: 'isMarketable', type: 'boolean' },
+                     { name: 'isMailUndeliverable', type: 'boolean' },
+                     { name: 'isTelephoneDisconnected', type: 'boolean' },
+                     { name: 'isDelisted', type: 'boolean' }
+                   ] }
+               ])
+    end
+
+  },
+
+  object_definitions: {
+
+    search_identity_resolution_input: {
+      fields: lambda do |_connection|
+        call('search_identity_resolution_input_schema')
+      end
+    },
+    search_identity_resolution_output: {
+      fields: lambda do |_connection|
+        call('identity_resolution_schema')
+      end
+    },
+    search_type_ahead_input: {
+      fields: lambda do |_connection|
+        call('search_type_ahead_input_schema')
+      end
+    },
+    search_type_ahead_output: {
+      fields: lambda do |_connection|
+        call('search_type_ahead_output_schema')
+      end
+    },
+    search_criteria_input: {
+      fields: lambda do |_connection|
+        call('search_criteria_input_schema')
+      end
+    },
+    search_criteria_output: {
+      fields: lambda do |_connection|
+        call('search_criteria_output_schema')
+      end
+    },
+    search_contact_input: {
+      fields: lambda do |_connection|
+        call('search_contact_input_schema')
+      end
+    },
+    search_contact_output: {
+      fields: lambda do |_connection|
+        call('search_contact_output_schema')
+      end
+    }
+
+  },
+
+  actions: {
+
+    search_identity_resolution: {
+      title: 'Search identity resolution',
+      subtitle: 'Search identity resolution in D&B',
+      description: "Search <span class='provider'>identity resolution</span> " \
+                   "in <span class='provider'>D&B</span>",
+      help: 'Identity Resolution returns the best matches for given' \
+      ' search criteria.',
+
+      input_fields: lambda do |object_definitions|
+        object_definitions['search_identity_resolution_input']
+      end,
+
+      execute: lambda do |_connection, input|
+        { Candidates: get('v1/match/cleanseMatch', input).
```

Comment:
Always use small letters for the start of variable. Please change the schema too.